src/languages/r.js

```diff
@@ -2,76 +2,148 @@
+            // `.Primitive`, minus those functions that are also keywords.
+            'abs acos acosh all any anyNA Arg as.call as.character' +
+            'as.complex as.double as.environment as.integer as.logical' +
+            'as.null.default as.numeric as.raw asin asinh atan atanh attr' +
+            'attributes baseenv browser c call ceiling class Conj cos cosh' +
+            'cospi cummax cummin cumprod cumsum digamma dim dimnames' +
+            'emptyenv exp expression floor forceAndCall gamma gc.time' +
+            'globalenv Im interactive invisible is.array is.atomic is.call' +
+            'is.character is.complex is.double is.environment is.expression' +
+            'is.finite is.function is.infinite is.integer is.language' +
+            'is.list is.logical is.matrix is.na is.name is.nan is.null' +
+            'is.numeric is.object is.pairlist is.raw is.recursive is.single' +
+            'is.symbol lazyLoadDBfetch length lgamma list log max min' +
+            'missing Mod names nargs nzchar oldClass on.exit pos.to.env' +
+            'proc.time prod quote range Re rep retracemem return round' +
+            'seq_along seq_len seq.int sign signif sin sinh sinpi sqrt' +
+            'standardGeneric substitute sum switch tan tanh tanpi tracemem' +
+            'trigamma trunc unclass untracemem UseMethod xtfrm',
         },
         relevance: 0
       },
-      {
-        // hex value
-        className: 'number',
-        begin: "0[xX][0-9a-fA-F]+[Li]?\\b",
-        relevance: 0
-      },
-      {
-        // explicit integer
-        className: 'number',
-        begin: "\\d+(?:[eE][+\\-]?\\d*)?L\\b",
-        relevance: 0
-      },
-      {
-        // number with trailing decimal
-        className: 'number',
-        begin: "\\d+\\.(?!\\d)(?:i\\b)?",
-        relevance: 0
-      },
-      {
-        // number
-        className: 'number',
-        begin: "\\d+(?:\\.\\d*)?(?:[eE][+\\-]?\\d*)?i?\\b",
-        relevance: 0
-      },
-      {
-        // number with leading decimal
-        className: 'number',
-        begin: "\\.\\d+(?:[eE][+\\-]?\\d*)?i?\\b",
-        relevance: 0
-      },
+
+      // Roxygen comments
+      hljs.COMMENT(
+        /#'/,
+        /$/,
+        {
+          contains: [
+            {
+              // Handle `@examples` separately to cause all subsequent code
+              // until the next `@`-tag on its own line to be kept as-is,
+              // preventing highlighting. This code is example R code, so nested
+              // doctags shouldn’t be treated as such. See
+              // `test/markup/r/roxygen.txt` for an example.
+              className: 'doctag',
+              begin: '@examples',
+              starts: {
+                contains: [
+                  { begin: /\n/ },
+                  {
+                    begin: /#'\s*(?=@[a-zA-Z]+)/,
+                    endsParent: true,
+                  },
+                  {
+                    begin: /#'\s?/,
```

Comment:
Is `\s?` or not important here?

Reply:
Actually it isn’t, it’s left over from a refactor. Removed!